api/v1alpha1/hostedcluster_types.go

```diff
@@ -32,6 +32,9 @@ type HostedClusterSpec struct {
 	// Networking contains network-specific settings for this cluster
 	Networking ClusterNetworking `json:"networking"`
 
+	// Autoscaling for compute nodes only, does not cover control plane
+	Autoscaling ClusterAutoscaling `json:"autoscaling"`
```

Comment:
should this `omitempty`?

Reply:
i wasn't sure, but given that all the options inside the struct are omitempty it probably makes sense here as well.